hrcase/hrcase.php

```diff
@@ -137,7 +137,6 @@ function hrcase_civicrm_buildForm($formName, &$form) {
       ),
     ));
     }
-    CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.hrcase', 'js/hrcase.js');
```

Comment:
why though? We are not removing any js files of unused extensions, why is this one singled out?

Reply:
Only the following extensions has been fixed or ignored
- Code is working fine
- Code is not working, but where the code is used is known to us and will be fixed in future tickets
- Extension is Disabled

But for `civicase`, extension is enabled. And upon further analysis it has been found that the code was written as part of https://issues.civicrm.org/jira/browse/HR-213, which is 4 years ago. Also its not found where the code has been used. Also the `case` menu item is not present anymore. 
So it has been removed.